.git-blame-ignore-revs

```diff
@@ -10,5 +10,8 @@ b88e1cd1b28e3609684c79a2ec0e88f26cfc362b
 b771971e3299c4fa56534b93421f7a2b9c7282fd
 9de88bb57ea9855da408cbec1dc8acb9079eda47
 8bc4688e52ea23ef688e283698f70a44388373eb
+0a5a9e803b56ec1bbd6232eff1c99dbbeef25eb7
+810cb346f05ac1aabfff931ab1a2b7b584add241
+5933b0018f8e29413e30dda9b906370d147bad45
 # Ran SystemTests and python/ctsm through black python formatter
 5364ad66eaceb55dde2d3d598fe4ce37ac83a93c
```

Comment:
I recognize most of this PR's mods as ones that I reviewed in #2081, which got merged in 2023. Typing
`git diff samsrabin/make-fsurdat-ace-dev ekluzek/simplemerge`
reveals a few diffs. In this file here, Erik's PR removes a line that we should probably keep.

Reply:
Yes, `540b256d1f3382f4619d7b0877c32d54ce5c40b6` should be added after line 17.

Reply:
Note that it's already present in main, so updating to the latest tag will bring it in, as long as conflicts are resolved correctly.